eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -121,6 +121,29 @@ stages:
         displayName: "Show Docker Logs"
         condition: succeededOrFailed()
 
+    - ${{ if startsWith(parameters.ServiceDirectory, 'resourcemanager') }}:
+      - job: MgmtMockTest
```

Comment:
It is probably worth creating a separate job template for this and having that imported based on the condition. Maybe move all of into mgmt-mock-test.yml and make that job template instead of step template.

Reply:
Good idea. I've reorg the files. BTW, it seems some template under templates/jobs belong to stages. Shall we need to move it properly at some point.

Reply:
It is just a convention so it isn't necessary to move exactly but it is a good idea to keep the templates split into the correct folder.